test/plugin/scenarios/dubbo-2.7.x-scenario/support-version.list

```diff
@@ -14,4 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-2.7.3
\ No newline at end of file
+2.7.3
+2.7.14
+2.7.15
```

Comment:
Do 2.7.14 and 2.7.15 have any difference to test?

Reply:
No, just because it's the latest version, I'll removed it.